libs/core/src/lib/select/select.component.ts

```diff
@@ -178,166 +194,320 @@ export class SelectComponent implements ControlValueAccessor, SelectInterface, O
 
     /** Select Input Mobile Configuration */
     @Input()
-    mobileConfig: MobileModeConfig = {hasCloseButton: true};
+    mobileConfig: MobileModeConfig = { hasCloseButton: true };
+
+    /** Popper.js options of the popover. */
+    @Input()
+    popperOptions: PopperOptions = {
+        placement: 'bottom-start',
+        modifiers: {
+            preventOverflow: {
+                enabled: true,
+                escapeWithReference: true,
+                boundariesElement: 'scrollParent'
+            }
+        }
+    };
```

Comment:
Feel free to remove it and its usages, `[options]` has been deprecated from popover and will be removed in future